packages/augur-ui/src/utils/convert-marketInfo-marketData.ts

```diff
@@ -13,6 +13,7 @@ import {
 import { convertUnixToFormattedDate } from './format-date';
 import { formatPercent, formatDai, formatNone, formatNumber, formatAttoRep } from './format-number';
 import { createBigNumber } from './create-big-number';
+import { keyBy } from './key-by';
```

Comment:
@bthaile 
Wasn't that renamed to `keyMarketInfoCollectionByMarketId`?

Reply:
gave a better name on later commit.